lint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: ./lint.sh path/to/html/source"
+  exit 1
+fi
+
+# show potential problems
+# note - would be nice if the ones with \s+ patterns actually cross lines, but, they don't...
+MATCHES=$(grep -ni 'xxx' $1 | perl -lpe 'print "\nPossible incomplete sections:" if $. == 1' \
+  & grep -niE '( (code|span|var)(>| data-x=)|[^<;]/(code|span|var)>)' $1 | perl -lpe 'print "\nPossible copypasta:" if $. == 1' \
```

Comment:
> Won't `&` cause the greps to run in parallel and change the order in a racy way?

Yes but in practice I think it’s not likely to actually affect the order. 

> A deliberate optimization?

Well I just didn’t see the need for the subsequent ones to wait but that said, in practice it doesn’t really speed anything up noticeably so it’s probably moot.

> Just `;` should make it serial.

Yeah I guess it’s safer to do that, especially given we’re not getting any speed up doing it otherwise. So please do go ahead and change it to that.